intelmq/bots/experts/jinja/expert.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-FileCopyrightText: 2021 Birger Schacht
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from intelmq.lib.bot import Bot
+
+import pathlib
+import os
+
+try:
+    from jinja2 import Template, TemplateError
+except ImportError:
+    Template = None
+
+
+class JinjaExpertBot(Bot):
+    """
+    Modify the message using the Jinja templating engine
+    Example:
+    fields:
+      output: The provider is {{ msg['feed.provider'] }}!
+      feed.url: "{{ msg['feed.url'] | upper }}"
+      extra.somejinjaoutput: file:///etc/intelmq/somejinjatemplate.j2
+    """
+
+    fields: dict = {}
+
+    def init(self):
+        if not Template:
+            raise ValueError("Library 'jinja2' is required, please install it.")
+
+        for field, template in self.fields.items():
+            if template.startswith("file:///"):
```

Comment:
What if the template is `file:///tmp/intelmq/{{ msg['feed.provider'] }}/{{msg['feed.name']}}}.txt`? :hankey: 
We could additionally check if the string contains curly braces (if actually someone uses file names with curly braces then he should rethink his file naming conventions)